from .exceptions import XPathValidationError
from .wrapper import validate_xpath

__all__ = ['validate_xpath', 'XPathValidationError']
